src/view/ShowIncomingVehiclesView.java

```diff
@@ -18,6 +18,7 @@ public class ShowIncomingVehiclesView extends JPanel implements ActionListener,
     private final ShowIncomingVehiclesViewModel showIncomingVehiclesViewModel;
     JLabel stationName;
     JTextArea stationIncomingVehicles;
+    JButton map;
```

Comment:
I recommend renaming this variable to be something like `mapButton`, since doing a variable called `map` when in reality, it's a button that LEADS to a map view is a bit misleading